app/src/test/java/edu/uc/hickmadc/customnotifications/ExampleUnitTest.kt

```diff
@@ -35,6 +35,6 @@ class ExampleUnitTest {
     fun confirmsSchedule_outputsSchedule () {
         val cal = Calendar.getInstance()
         var scheduleTest: Schedule = Schedule(cal, 5, 1)
-        assertEquals(cal, scheduleTest.original_occurence);
+        assertEquals(cal, scheduleTest.originalOccurance);
```

Comment:
In the unit test, I had to change the spelling of the originalOccurance variable after I had changed it to camelcase in the data class.

Reply:
Good.